active/0000-ember-data-reference-unification.md

```diff
@@ -0,0 +1,362 @@
+  is present as `reference.value`. Otherwise, the value of this
+  property is `null`.
+
+  @property
+*/
+```
+
+### `data`
+
+```js
+/**
+  The value of the (normalized) representation of this entity. For
+  example, `recordReference.data` will return a normalized dictionary
+  of attributes and links.
+
+  @property
+*/
+```
+
+### `metadata`
+
+```js
+/**
+  The most recent value of the metadata returned by the server for
+  the value represented by this reference.
+
+  @property
+*/
+```
+
+### `load`
+
+```js
+/**
+  Triggers a fetch for the backing entity based on its `remoteType`
+  (see `remoteType` definitions per reference type).
+
+  @method
+  @param {Object} an options hash, similar to the one currently
+    passed to `store.find`.
+*/
+```
+
+### `unload`
+
+```js
+/**
+  Unload the entity referred to by this relationship. After this
+  operation, its `value`, `data` and `metadata` members will return
+  to `null`, and the record itself will be purged from the identity
+  map.
+
+  @method
+*/
+```
+
+## RecordReference
+
+### `remoteType`
+
+```js
+/**
+  How the reference will be looked up with it is loaded:
+
+  * `link`, a URL
+  * `identity`, by the `type` and `id`
+*/
+```
+
+### `type`
+
+```js
+/**
+  The type of the record that this reference refers to.
+
+  @property
+*/
+```
+
+### `id`
+
+```js
+/**
+  The `id` of the record that this reference refers to.
+
+  Together, the `type` and `id` properties form a composite key
+  for the identity map.
+
+  @property
+*/
+```
+
+## HasManyReference
+
+### `remoteType`
+
+```js
+/**
+  How the reference will be looked up when it is fetched:
+
+  * `link`, a URL provided by the server
+  * `ids`, a list of IDs provided by the server
+  * `dynamic`, a dynamic URL will be created based on the identity
+    of the parent.
+
+  @property
+*/
+```
+
+### `link`
+
+```js
+/**
+  If the `remoteType` is `link`, the URL to use to load the relationship.
+
+  @property
+*/
+```
+
+### `ids`
+
+```js
+/**
+  If the `remoteType` is `ids`, a list of IDs that is used to formulate
+  the query to the server (together with `type`).
+
+  @property
+*/
+```
+
+### `type`
+
+```
+/**
+  The model type represented by this relationship.
+
+  @property
+*/
+```
+
+### `parent`
+
+```js
+/**
+  A reference to the record that has this `hasMany` on it.
+
+  @property
+*/
+```
+
+### `inverse`
+
+```js
+/**
+  If there is an inverse relationship, this property is a reference
+  to it.
+
+  @property
+*/
+```
+
+## BelongsToReference
+
+### `remoteType`
+
+```js
+/**
+  How the reference will be looked up when it is fetched:
+
+  * `link`, a URL provided by the server
+  * `id`, an ID to use to form the URL
+  * `dynamic`, a dynamic URL will be created based on the identity
+    of the parent.
+
+  @property
+*/
+```
+
+### `link`
+
+```js
+/**
+  If the `remoteType` is `link`, the URL to use to load the relationship.
+
+  @property
+*/
+```
+
+### `ids`
```

Comment:
i think this should be `id` as it is a belongsTo and will only have one id, thus not being plural ids